src/sentry/models/organizationmemberinvite.py

```diff
@@ -77,6 +77,9 @@ class OrganizationMemberInvite(DefaultFieldsModel):
     __relocation_scope__ = RelocationScope.Organization
 
     organization = FlexibleForeignKey("sentry.Organization", related_name="invite_set")
+    # SCIM provisioning requires that the OrganizationMember object exist. Until the user
+    # accepts their invite, the OrganizationMember is a placeholder and will not be surfaced via API.
+    organization_member = FlexibleForeignKey("sentry.OrganizationMember")
```

Comment:
Just wanted to check - is there a way to not have the dummy row? Or is that just a requirement?

Reply:
we need to return the member id via SCIM, which should be consistent across invite and when the member accepts the invite :/